src/ui/MainRootWindow.qml

```diff
@@ -78,7 +78,7 @@ ApplicationWindow {
     QtObject {
         id: globals
 
-        readonly property var       activeVehicle:                  QGroundControl.multiVehicleManager.activeVehicle
+        readonly property var       activeVehicle:                  QGroundControl.multiVehicleManager.activeVehicle ? QGroundControl.multiVehicleManager.activeVehicle : QGroundControl.multiVehicleManager.offlineEditingVehicle
```

Comment:
Haven't had time to look at this fully yet. But quick glance shows this is an incorrect change. The activeVehicle global only represents a truly connected vehicle. It i never supposed to be connected to the offline vehicle.

Reply:
I wasn't sure about that one, makes sense though. I reverted that part in latest commit.